plugins/ltac/coretactics.mlg

```diff
@@ -197,18 +185,23 @@ TACTIC EXTEND esplit_with
   }
 END
 
+{
+
+let exists with_evars bll =
```

Comment:
You should probably move this definition to `tactics.ml`. It is not recommended to have code in mlg, despite the current situation.

Reply:
Done.